tests/org.jboss.tools.usage.ui.bot.test/src/org/jboss/tools/usage/ui/bot/test/CloseUsageWindowTest.java

```diff
@@ -0,0 +1,17 @@
+package org.jboss.tools.usage.ui.bot.test;
```

Comment:
Please add copyright header and author and small description of the testing class.